docs/upgrading/index.rst

```diff
@@ -11,6 +11,16 @@ However, upgrade support should be considered alpha at this time; it has not
 been extensively tested on production clusters. Please use with caution and
 report any issues you have with the process.
 
+Upgrading OS packages
+---------------------
+
+We provide two playbooks for upgrading OS-level system packages on a cluster
+after is has been provisioned: ``playbooks/upgrade-packages.yml`` and
```

Comment:
"after **it** has been provisioned". But, maybe we just remove this phrase to avoid confusion? I am not sure how consistent we are but, in other places in the docs, we use the term "provision" to refer to the process of running Ansible on the cluster. Only the rolling upgrade would be safe on a "provisioned" cluster in that case.